play-ws-standalone-json/src/test/scala/play/api/libs/ws/JsonBodyReadablesSpec.scala

```diff
@@ -12,7 +12,7 @@ import play.api.libs.json.{ JsSuccess, JsValue }
 
 class JsonBodyReadablesSpec extends Specification with MustMatchers {
 
-  class StubResponse(byteArray: Array[Byte]) extends StandaloneWSResponse {
+  class StubResponse(byteArray: Array[Byte], charsetName: String = "UTF-8") extends StandaloneWSResponse {
```

Comment:
Use `java.nio.charset.Charset` + `java.nio.charset.StandardCharsets` instead. Harder to make mistakes.